src/optimizer/rules/implementation_rules.cpp

```diff
@@ -127,7 +127,7 @@ void LogicalGetToPhysicalIndexScan::Transform(common::ManagedPointer<OperatorExp
         if (IndexUtil::SatisfiesSortWithIndex(accessor, sort_prop, get->GetTableOid(), index)) {
           std::vector<AnnotatedExpression> preds = get->GetPredicates();
           auto op = std::make_unique<OperatorExpression>(
-              IndexScan::Make(db_oid, ns_oid, index, std::move(preds), get->GetTableAlias(), is_update,
+              IndexScan::Make(db_oid, ns_oid, get->GetTableOid(), index, std::move(preds), is_update,
```

Comment:
In theory we don't need ns_oid anymore if oids are unique to a database, right? I believe we can uniquely identify tables and indexes within a database with just their oid. It's not urgent, and likely just legacy behavior from Peloton when things had to be fully qualified.